packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -612,6 +612,87 @@ class CustomScrollView extends ScrollView {
   });
 
   /// The slivers to place inside the viewport.
+  ///
+  /// ## What is a sliver?
+  ///
+  /// A _sliver_ is a widget backed by a [RenderSliver] subclass, i.e. one that
+  /// implements the constraint/geometry protocol that uses [SliverConstraints]
+  /// and [SliverGeometry].
```

Comment:
The reason I put it here is because it is the target of the "slivers" cross-reference in [CustomScrollView].
I'll add links to it from various places.